### PR TITLE
issue_137 update the docker compose settings

### PR DESCRIPTION
**Pull request by zqian**

### CODE REVIEW

File: SIS_import/docker-compose.yml [hunk 1]
```diff
@@ -13,4 +13,4 @@ services:
```
Comment (pushyamig) [inline SIS_import/docker-compose.yml:13]: Optional: The docker compose latest version seems to be 3.8. currently it is given 3 which will be defaults to 3.0

https://docs.docker.com/compose/compose-file/compose-versioning/#version-3

File: SIS_import/Dockerfile [hunk 2]
```diff
@@ -34,4 +32,6 @@ RUN mkdir /usr/src/app/data
 RUN chown -R root:root /usr/src/app/data
 RUN chmod g+rw /usr/src/app/data
 
+ENV PROCESS=sis_set_url
```
Comment (pushyamig) [inline SIS_import/Dockerfile:35]: Why is this needed?

**Comment by pushyamig** [comment]
I was able to build successfully and not with out errors as provided in the problem statement.

**Comment by pushyamig** [comment]
According to Docker

**Comment by pushyamig** [comment]
Sorry, accidentally closed the issue, but i reopened now

File: SIS_import/docker-compose.yml (hunk 1 above)
Comment (pushyamig) [inline SIS_import/docker-compose.yml:13]: The docker file has a configuration that says ` ${HOME}/secrets/sis:/app/config/secrets` so I put the `.env` in ` home directory sis folder` but when I ran the script saying `docker compose up` it complained saying the `.env` is not in the project root directory. Just wondering why i see this problem ?

**Comment by pushyamig** [comment]
The `docker compose up` is kept on running for a long time with canvas beta configuration, so this is possibly expected. So I Stopped the process. To me this passes end-to-end testing. is this acceptable level of testing?

**Review by pushyamig — COMMENTED** [review]
I feel this change works but I have some comments left for clarification purposes since i ran this project long time ago

File: SIS_import/Dockerfile (hunk 2 above)
Comment (zqian) [reply SIS_import/Dockerfile:35]: the PROCESS variable is used inside [rclone_setup.sh](https://github.com/tl-its-umich-edu/canvas/blob/master/SIS_import/rclone_setup.sh), where it defaults to value`sis_set_url ` if not set. 

I added it here to demonstrate that this is a configurable ENV variable inside the Dockerfile.

File: SIS_import/docker-compose.yml (hunk 1 above)
Comment (zqian) [reply SIS_import/docker-compose.yml:13]: Thanks! The docker version has been upgraded to v3.8

**Review by pushyamig — APPROVED** [review]
This build and ran fine.

File: SIS_import/docker-compose.yml (hunk 1 above)
Comment (zqian) [reply SIS_import/docker-compose.yml:13]: I have reported this as a separate issue: #141
